flask-server/avenv/lib/python3.12/site-packages/flask/sessions.py

```
@@ -0,0 +1,399 @@
+    def setdefault(self, key: str, default: t.Any = None) -> t.Any:
+        self.accessed = True
+        return super().setdefault(key, default)
+
+
+class NullSession(SecureCookieSession):
+    """Class used to generate nicer error messages if sessions are not
+    available.  Will still allow read-only access to the empty session
+    but fail on setting.
+    """
+
+    def _fail(self, *args: t.Any, **kwargs: t.Any) -> t.NoReturn:
```

Ruff (0.13.2)

Unused method argument: `args`

(ARG002)